### PR TITLE
Command Line Entry for Check

### PR DESCRIPTION
**Pull request by moluwole**
To use the command entry point, 

1. Run `python setup.py develop`
2. to use, `binaryornot filename`

### CODE REVIEW

File: binaryornot/check.py [hunk 1]
```diff
@@ -31,3 +32,21 @@ def is_binary(filename):
     # Check if the starting chunk is a binary string
     chunk = get_starting_chunk(filename)
     return is_binary_string(chunk)
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Check if a "
+                                                 "file passed as argument is "
+                                                 "binary or not")
```
Comment (pombredanne) [inline binaryornot/check.py]: What about a simpler: `Check if a file is binary or not.`

File: binaryornot/check.py [hunk 2]
```diff
@@ -31,3 +32,21 @@ def is_binary(filename):
...
+
+    parser.add_argument("filename", help="File name to check for. If "
+                                         "the file is not in the same folder, "
+                                         "include full path")
+
```
Comment (pombredanne) [inline binaryornot/check.py]: What about using a simpler `"File name or path to check."`

File: binaryornot/check.py [hunk 3]
```diff
@@ -8,8 +8,9 @@
 """
 
 import logging
+import argparse
```
Comment (pombredanne) [inline binaryornot/check.py]: What about sorting these imports?

File: binaryornot/check.py [hunk 4]
```diff
@@ -31,3 +32,21 @@ def is_binary(filename):
...
+                                                 "file passed as argument is "
+                                                 "binary or not")
+
+    parser.add_argument("filename", help="File name to check for. If "
+                                         "the file is not in the same folder, "
+                                         "include full path")
+
+    args = parser.parse_args()
+
+    print(is_binary(**vars(args)))
```
Comment (pombredanne) [inline binaryornot/check.py]: This will print True or False literally, right? Is this the best way?

File: requirements_dev.txt [hunk 5]
```diff
@@ -6,3 +6,4 @@ Sphinx
 coverage
 tox
 hypothesis
+argparse
```
Comment (pombredanne) [inline requirements_dev.txt]: Why would this be needed? This is builtin since 2.7 and 3.2

File: binaryornot/check.py (hunk 4 above)
Comment (pydanny) [reply binaryornot/check.py]: What's a better response?

File: binaryornot/check.py (hunk 3 above)
Comment (pydanny) [reply binaryornot/check.py]: Agreed. That's a nice touch for projects.

**Pull request merged**